mesonbuild/backend/backends.py

```diff
@@ -147,6 +148,13 @@ def __init__(self, fname: str, outdir: str, outdir_name: str, aliases: T.Dict[st
         self.optional = optional
         self.tag = tag
 
+class InstallEmptyDir:
+    def __init__(self, path: str, install_mode: 'FileMode', subproject: str, tag: T.Optional[str] =None):
```

Comment:
```suggestion
    def __init__(self, path: str, install_mode: 'FileMode', subproject: str, tag: T.Optional[str] = None):
```

Missing a space after the =